### PR TITLE
RD-3910: Blocked passing unnecessary data upon navigating to deployment page

### PR DESCRIPTION
**Pull request by NorbertAmartus**
## Description

[Issue](https://cloudifysource.atlassian.net/browse/RD-3910) is appearing due to unnecessary passing `nodeId` to deployment page.
It is read by `events` widget, making the event list to be incorrectly filtered.

### More context
That behavior seems to be related to the fact, that while clicking on the "Go to deployment page" button (inside `topology` widget) we are not only executing function attached to that button, but also the one responsible for handling selection of a specific node (to which the clicked button is attached 😅)
As handler for node selection is putting some extra data into the application `context`, we are getting into the situation when we are trying to navigate to the deployment page with a data that shouldn't be passed.
It looks like an issue related to the event bubbling - coming from the [topology](https://github.com/cloudify-cosmo/cloudify-blueprint-topology).

## Screenshots / Videos

N/A

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

### 1 attempt (on **all** test scenarios)
[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3199/)
`topology_spec` is failing

### 2 attempt (on `topology_spec`)
[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3200/)

## Documentation

N/A

### CODE REVIEW

**Review by qooban — COMMENTED** [review]
Looks fine. Added just one comment.

File: app/translations/en.json [hunk 1]
```diff
@@ -1268,6 +1268,34 @@
...
+                        "Blueprint",
+                        "Deployment",
+                        "Deployment ID",
+                        "Workflow",
+                        "Operation",
+                        "Node ID",
+                        "Node instance ID",
+                        "Message"
+                    ],
+                    "default": "Icon,Timestamp,Blueprint,Deployment,Workflow,Operation,Node ID,Node instance ID,Message"
```
[the remark this answers is not in the document]
Comment (NorbertAmartus) [reply app/translations/en.json:1290]: Sure, seems like a valid idea 😉

[the remark this answers is not in the document]
Comment (NorbertAmartus) [reply app/translations/en.json]: For now, `fieldsToShow` `default` value has been transfered to `widget` declaration file 🎸

[the remark this answers is not in the document]
Comment (qooban) [reply app/translations/en.json]: As I wrote in my original comment I had in mind a similar solution as in Blueprints or Blueprint Catalog widget 🙂 
Default value should not have direct translation IMHO as it is just a list of strings already translated.

See: https://github.com/cloudify-cosmo/cloudify-stage/blob/9d3dff86fcff96d5b4a944f52fd83ccf846d3c19/widgets/blueprints/src/widget.tsx#L32 or https://github.com/cloudify-cosmo/cloudify-stage/blob/9d3dff86fcff96d5b4a944f52fd83ccf846d3c19/widgets/blueprintCatalog/src/widget.tsx#L78

WDYT?

[the remark this answers is not in the document]
Comment (NorbertAmartus) [reply app/translations/en.json]: Oh, I interpretated it in the another way 😅
Having that approach in mind, it makes totally sense 💪 

Did something similar, but not quite the same, as the current **default** values are not the same as the **available** values

Thanks for putting my interpretation into the right direction 😁🎸

**Review by qooban — APPROVED** [review]
LGTM 👍🏻

File: app/translations/en.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (qooban) [reply app/translations/en.json]: Thanks for addressing it 👍🏻

**Pull request merged**